Redshift profile iam fix (#5691)
dbt-labs/docs.getdbt.com: website/docs/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb.md

<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/postgres-redshift-connection.png" width="70%" title="Configuring a Redshift connection"/>

### Authentication Parameters

For authentication, dbt Cloud users can use either a **Database username and password**, or they can now use **IAM User authentication** to Redshift via [extended attributes](/docs/dbt-cloud-environments#extended-attributes).

<Tabs
  defaultValue="database"
  values={[
    {label: 'Database', value: 'database'},

>     {label: 'IAM User', value: 'iam-user-inline'}

Reviewer: ```suggestion
    {label: 'IAM User', value: 'iam-user-inline'},
```

<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/postgres-redshift-connection.png" width="70%" title="Configuring a Redshift connection"/>

> For **authentication**, dbt Cloud users can pick Database username and password. In addition, it is now possible to use IAM User authentication to Redshift, via [extended attributes](/docs/core/connect-data-platform/redshift-setup#authentication-parameters).

Reviewer: ```suggestion
For authentication, dbt Cloud users can use either a Database username and password, or they can now use IAM User authentication to Redshift via [extended attributes](/docs/core/connect-data-platform/redshift-setup#authentication-parameters).
```
